### PR TITLE
cleanup(hacks): just some reodering of lines to follow the template

### PR DESCRIPTION
**Pull request by pvinis**
The type of this PR is: **cleanup**

This PR resolves [CX-]

### Description

After the RFC https://github.com/artsy/README/issues/390, I wanted to clean up this file a bit, so it's a better example.

Now all entries have a name, a time to remove, and the context of the hack, in that order.

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [ ] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434

### CODE REVIEW

File: HACKS.md [hunk 1]
```diff
@@ -85,34 +85,32 @@
...
   - focused/error border states
   - shrinking the icon size to work nicely with our inputs
   - aligning inner inputs nicely
 
-  We can remove these hacks once we switch to Stripe's forthcoming official react-native library.
-
 - react-native-mapbox-gl/maps - 8.2.0-beta2 postinstall script
 
-  We had issues with our archive becoming invalid and failing to export when we updated mapbox and cocoapods
+  Now.
```
Comment (brainbicycle) [inline HACKS.md:111]:
```suggestion
  We can remove these hacks NOW.
```

**Review by brainbicycle — APPROVED** [review]
Thanks for the clean-up, small suggestion for the one we can remove now, feel free to merge when ready.

**Comment by pvinis** [comment]
Ok this should be way cleaner now. I realized that I was mostly using this file as a text file, and many others as a rendered md file, so I ended up changing the format a bit so that it's clean and clear as both a text file and a md file.

**Comment by damassi** [comment]
Once small additional suggestion would be to [create a new `Hacks` section](https://github.com/artsy/force/pull/7767/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R18) in the main README meta area. I've added this to force.

**Pull request merged**